from lamb.exceptions import LambException


class ContextException(LambException):
    pass
